src/main/java/com/example/tournament/Service/TournamentService.java

```diff
@@ -0,0 +1,213 @@
+package com.example.tournament.Service;
+
+
+import com.example.tournament.DTO.TournamentDto;
+import com.example.tournament.Entity.Match;
+import com.example.tournament.Entity.Team;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+import java.util.Scanner;
+
+@Service
+public class TournamentService {
+    private List<Team> teamList = new ArrayList<>();
+    private List<Match> matchList = new ArrayList<>();
+    private ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
this can be injected. Remember: class don't want to know how to initialize it's dependencies